Makefile.omdev.mingw

```diff
@@ -113,7 +113,10 @@ omoptim: omc omplot qtclientsDLLs
 	$(MAKE) -f $(defaultMakefileTarget) -C OMOptim OMBUILDDIR=$(OMBUILDDIR) CC="$(CC)" CXX="$(CXX)"
 
 omsens:
-	cp -rfp OMSens $(OMBUILDDIR)
+	$(MAKE) -f $(defaultMakefileTarget) -C OMSens OMBUILDDIR=$(OMBUILDDIR)
```

Comment:
Do you need to pass `CC="$(CC)" CXX="$(CXX)"` to the next makefile target, so that the Windows compilation is using the correct CC?

Reply:
No. The makefile just compiles some fortran files, convert them to python files and then build a python binary.

However, I should make sure that `omsens` target is called during the `Win/MinGW` stage.